README.md

```diff
@@ -12,6 +12,8 @@ $ bower install gaia-components/gaia-component
 
 ```js
 var MyComponent = component.register('my-component', {
+  // extends from parent element could inherit parent element's behavior
+  extends: HTMLButtonElement.prototype,
```

Comment:
This comment is quite vague. I think we should be clear that the only thing `extends` does is extend the given `prototype` nothing more (unfortunately :cry:)